src/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: install
+install:
+	pip install -r requirements_dev.txt
+	pre-commit install
+
+.PHONY: sync
+sync:
+	pip-sync requirements.txt requirements_dev.txt
+
+.PHONY: requirements
+requirements: requirements.in
+	pip-compile -v --output-file requirements.txt requirements.in
+
+.PHONY: upgrade
+upgrade:
+	pur --patch=Django -r requirements.in
+	pur --no-recursive -r requirements_dev.txt
+	pip-compile -v --upgrade --output-file requirements.txt requirements.in
```

Comment:
Misschien nog `make release` toevoegen die `publish` doet? of dat wordt alleen door jenkins gebruikt?

Reply:
Dat is wel handig, voeg ik toe! Want, je kunt straks ook gewoon direct vanuit deze repo publishen, dat gaat een stuk sneller dan via Jenkins.

Reply:
Gaat dat dan naar acceptatie of productie ?